curp/src/server/raw_curp/mod.rs

```diff
@@ -776,6 +784,57 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
             ReadState::Ids(IdSet::new(ids))
         }
     }
+
+    /// Handle `move_leader`
+    pub(super) fn handle_move_leader(&self, node_id: ServerId) -> Result<bool, CurpError> {
```

Comment:
Please try to give `node_id` a more meaning name, such as `target_id` or `target_node_id`.